src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -253,7 +272,37 @@ export default class VictoryScatter extends React.Component {
     return Helpers.evaluateStyle(baseDataStyle, data);
   }
 
-  renderPoint(data, index, calculatedProps) {
+  static getKeyedData(data) {
+    return data.reduce((keyedData, datum, idx) => {
```

Comment:
I think all this stuff belongs in victory-core helpers since it will also be used in victory-pie etc.